impl/c-qube/src/utils/hash.spec.ts

```diff
@@ -1,7 +1,28 @@
+import yargs from 'yargs';
 import { hash, unhash } from './hash';
 import * as crypto from 'crypto';
+import { hideBin } from 'yargs/helpers';
+import { parseArguments } from './cli';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 
+jest.mock('yargs', () => ({
```

Comment:
why are yargs tests inside hash.spec.ts? what is the relevance for adding them here?